### DISCUSSION TITLE
Make the ACME plugin works at service level.

### DISCUSSION THREAD

**Discussion by cybercoder** [Kong/kong#12619] [Help] [answered]
I've commented the line [135](https://github.com/Kong/kong/blob/ab7232ea98be93f1c4a69482aa9995808df6819a/kong/plugins/acme/schema.lua#L135) of the plugin, and now it creates multiple ACMEs per services. But it is not working without any error.
Any Idea?

**Comment by chronolaw** [comment]
Could you give more details about the kong gateway? such as version/config and others.

**Reply by cybercoder** [reply]
the official community's latest version on docker with pg db.

**Reply by chronolaw** [reply]
If you provide configuration  and re-produce steps it will be very helpful to trouble shooting.

**Reply by cybercoder** [reply]
There are no specific configurations. I just need to enable the ACME plugin per service. It's just working globally.

**Reply by chronolaw** [reply]
I think that you should give us at least one minimal re-produce case, then we can understand what happened.
Or some error logs?

**Reply by fffonion** [reply, marked as answer]
@cybercoder There're some global state, if you have multiple acme instances in kong, they will overwrite each other. You may not notice error but there're race condition.
